roles/as_code/lib/shared_expectations/live_services_expectations.rb

```diff
@@ -0,0 +1,10 @@
+class LiveServicesExpectations < JobSpec::Role::Expectations
+  expected 'to provide an outstanding customer service',
+    'We expect our Live Services team to be a delightful people to talk to for our customers. Our conversations should be professional and friendly, we should small talk where possible, use a smiley face once in a while and generally be nice folk.'
```

Comment:
interact with over talk? IDK if talk implies phone/Hangout, and actually, much of our CS interaction is written